### ISSUE TITLE
reset version of a library on all packages

### ISSUE THREAD

**Issue by jogelin** [guigrpa/oao#28]
Hello,

Sometimes, in a monorepo, it could be good to change the version of a library in all packages. For example, if packages uses Angular or React, we would like to be sure that all packages are sharing the same version.

I think there are different ways of doing :
## Having a command that set a version for a dependency, for example:
 ```
$ oao set-version @angular 4.0.0 --> all dependencies of this scope are updated
```
```
$ oao set-version @angular/core,@angular/http 4.0.0 --> only core and http are updated
```
## Copy versions from root/parent :
In my case it is what I am doing, I don't like my solution :

**BEFORE**
**`package.json`:**
```
{
    "name": "root",
    "version":"0.1.0"
    "peerDependenciesPlaceholders": {
        "0.0.0-ANGULAR_VERSION_PLACEHOLDER": "^4.0.0-rc.1"
    }
}
```
**`packages/lib1/package.json`:**
```
{
    "name": "lib-1",
    "version":"0.1.0",
    "peerDependencies": {
        "@angular/core": "0.0.0-ANGULAR_VERSION_PLACEHOLDER",
    }
}
```

**RUN INTERNAL SCRIPT**
I run internal script that replaces placeholders on all packages corresponding to the placeholders found in `peerDependenciesPlaceholders` of the root `package.json`

**AFTER**
**`packages/lib1/package.json`:**
```
{
    "name": "lib-1",
    "version":"0.1.0",
    "peerDependencies": {
        "@angular/core": "^4.0.0-rc.1"
    }
}
```

**Comment by jogelin** [comment]
ok maybe the upgrade will do the job, you can confirme ?

sorry ;-)

**Comment by jogelin** [comment]
I could use upgrade in this case if
- I can apply a version of a dependency on all packages (not one command per package)
- I can apply upgrade on peer dependencies

**Comment by guigrpa** [comment]
Autumn cleaning — please reopen if there's still interest.

**Comment by guigrpa** [comment]
Shipped in v1.5.0!
